src/main/kotlin/org/openrewrite/RecipeMarkdownGenerator.kt

```diff
@@ -580,11 +589,21 @@ class RecipeMarkdownGenerator : Runnable {
                         ```
                         {% endcode %}
                         {% endtab %}
+    
+                        {% tab title="Maven Command Line" %}
+                        {% code title="shell" %}
+                        ```shell
+                        mvn org.openrewrite.maven:rewrite-maven-plugin:$mavenPluginVersion:run \
+                          -Drewrite.recipeArtifactCoordinates=${origin.groupId}:${origin.artifactId}:${origin.version} \
+                          -DactiveRecipes=${recipeDescriptor.name}
+                        ```
+                        {% endcode %}
+                        {% endtab %}
                         {% endtabs %}
                         
                     """.trimIndent())
                 }
-                writeln("Recipes can also be activated directly from the command line by adding the argument `-Drewrite.activeRecipes${recipeDescriptor.name}`")
+                writeln("Recipes can also be activated directly from the command line by adding the argument `-Drewrite.activeRecipes=${recipeDescriptor.name}`")
```

Comment:
At the very least this change should make it in, similar to https://github.com/openrewrite/rewrite-recipe-markdown-generator/commit/a43d3fcbf0f55a272fa19c6b8022be908431dcd5 (@tkvangorder ).